common.js

```diff
@@ -0,0 +1,13 @@
+; 'use strict';
+
+const app = {};
+
+app.methods = {};
```

Comment:
seems not used

Reply:
app is used, but not app.methods, removing it